cypress/e2e/smoke/create_tx.cy.js

```diff
@@ -15,8 +15,13 @@ describe('Queue a transaction on 1/N', () => {
   })
 
   it('should create and queue a transaction', () => {
+    // Assert that "New transaction" button is visible
+    cy.contains('New transaction', {
+      timeout: 60_000, // `lastWallet` takes a while initialize in CI
```

Comment:
This is horribly long but 30s was sometimes too short. It doesn't seem to need this long but I'd sooner have it longer than necessary. As soon as the button is present, it continues.

Reply:
Why the long timeout in this assertion? There is no request before this modal opens. 
If this is a bottleneck disregard my comment -> I thought the timeout would come only with the `/estimations` request